pkg/platform/kube/functionres/lazy.go

```diff
@@ -1837,13 +1841,20 @@ func (lc *lazyClient) addIngressToSpec(ctx context.Context,
 			return errors.Wrap(err, "Failed to format ingress pattern")
 		}
 
-		httpIngressPath := extv1beta1.HTTPIngressPath{
-			Path: formattedPath,
-			Backend: extv1beta1.IngressBackend{
-				ServiceName: kube.ServiceNameFromFunctionName(function.Name),
-				ServicePort: intstr.IntOrString{
-					Type:   intstr.String,
-					StrVal: ContainerHTTPPortName,
+		// TODO: make pathType configurable - via ui or annotation
```

Comment:
ui means function spec.
the TODO should be via function (or trigger) annotation or platform configuration, which holds some default function values.